qmxgraph/page/graphs.js

```diff
@@ -105,6 +105,12 @@ graphs.createGraph = function createGraph (container, options, styles) {
         return cellId.indexOf(mxCell._PORT_ID_PREFIX) === 0;
     };
 
+    mxCell.prototype.isSelectable = function() {
+        var styleName = this.getStyle();
```

Comment:
If you move this logic to `mxGraph.prototype.isCellSelectable` you could use `this` a graph reference and not using `graph` from outer scope.

Reply:
Done.